carbon_black_cloud/plugin.spec.yaml

```
@@ -0,0 +1,456 @@
+      title: "Rooted by Analytics"
+      type: boolean
+      description: "Rooted by analytics"
+      required: false
+    rootedByAnalyticsTime:
+      title: "Rooted by Analytics Time"
+      type: string
+      description: "Rooted by analytics time"
+      required: false
+    rootedBySensor:
+      title: "Rooted by Sensor"
+      type: boolean
+      description: "Rooted by sensor"
+      required: false
+    rootedBySensorTime:
+      title: "Rooted by Sensor Time"
+      type: string
+      description: "Rooted by Sensor Time"
+      required: false
+    scanLastActionTime:
+      title: "Scan Last Action Time"
+      type: integer
+      description: "Scan last action time"
+      required: false
+    scanLastCompleteTime:
+      title: "Scan Last Complete Time"
+      type: integer
+      description: "Scan last complete time"
+      required: false
+    scanStatus:
+      title: "Scan Status"
+      type: string
+      description: "Scan status"
+      required: false
+    sensorKitType:
+      title: "Sensor Kit Type"
+      type: string
+      description: "Sensor kit type"
+      required: false
+    sensorOutOfDate:
+      title: "Sensor Out of Date"
+      type: boolean
+      description: "Sensor out of date"
+      required: false
+    sensorPendingUpdate:
+      title: "Sensor Pending Update"
+      type: boolean
+      description: "Sensor pending update"
+      required: false
+    sensorStates:
+      title: "Sensor States"
+      type: "[]string"
+      description: "Sensor states"
+      required: false
+    sensorVersion:
+      title: "Sensor Version"
+      type: string
+      description: "Sensor version"
+      required: false
+    status:
+      title: "Status"
+      type: string
+      description: "Status"
+      required: false
+    targetPriorityType:
+      title: "Target Priority Type"
+      type: string
+      description: "Target priority type"
+      required: false
+    testId:
+      title: "Test ID"
+      type: integer
+      description: "Test ID"
+      required: false
+    uninstallCode:
+      title: "Uninstall Code"
+      type: string
+      description: "Uninstall code"
+      required: false
+    uninstalledTime:
+      title: "Uninstalled Time"
+      type: string
+      description: "Uninstalled time"
+      required: false
+    vdiBaseDevice:
+      title: "VDI Base Device"
+      type: string
+      description: "VDI Base Device"
+      required: false
+    virtualMachine:
+      title: "Virtual Machine"
+      type: boolean
+      description: "Virtual machine"
+      required: false
+    virtualizationProvider:
+      title: "Virtualization Provider"
+      type: string
+      description: "Virtualization Provider"
+      required: false
+    windowsPlatform:
+      title: "Windows Platform"
+      type: string
+      description: "Windows platform"
+      required: false
+
+connection:
+  api_id:
+    title: API ID
+    type: string
+    description: API ID
+    example: ABCDE12345
```

```suggestion
    example: ABCDE12345
```
Is there anything more realistic we can use?